src/renderer/components/dialogs/CreateChat-Styles.js

```diff
@@ -105,11 +105,11 @@ export const GroupImageWrapper = styled.div`
 `
 
 export const GroupImage = (props) => {
-  const { groupImage, onSetGroupImage, onUnsetGroupImage } = props
+  const { groupImage, onSetGroupImage, onUnsetGroupImage, ...otherProps } = props
   return (
     <GroupImageWrapper>
       { groupImage && <AvatarImage src={groupImage} onClick={onSetGroupImage} {...props} /> }
-      { !groupImage && <AvatarBubble onClick={onSetGroupImage} {...props}>G</AvatarBubble> }
+      { !groupImage && <AvatarBubble onClick={onSetGroupImage} {...otherProps}>G</AvatarBubble> }
```

Comment:
not sure if we should use such a wildcard

Reply:
Well before it passed all properties - now all except onSetGroupImage, onUnSetGroupImage, GroupImage where onSetGroupImage, onUnSetGroupImage are not known properties for AvatarBubble...

Reply:
ok